.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -53,12 +53,15 @@ merge-dev-secrets:
     master_branch_name: master
   rules:
     - !reference [ .filter-non-nightly-docker-updates-rule, rules ]
-    - if: '$CI_PIPELINE_SOURCE =~ /^(push)$/ && $CI_COMMIT_BRANCH == $master_branch_name'
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH == $master_branch_name'
   stage: unittests-and-validations
   script:
+    - EXIT_CODE=0
     - !reference [.download-demisto-conf]
-    - section_start "Merging and deleting dev secrets" --collapsed
-    - python3 ./Tests/scripts/merge_and_delete_dev_secrets.py -sa "$GSM_SERVICE_ACCOUNT" --gsm_project_id_dev "$GSM_PROJECT_ID_DEV" --gsm_project_id_prod "$GSM_PROJECT_ID" >> "${ARTIFACTS_FOLDER}/logs/merge_secrets.log"
+    - section_start "Merging and deleting dev secrets"
```

Comment:
Why not collapsed?

Reply:
because it's the "main" section for this build job. and it's output should be shown